pkg/pipelines/pipelines.go

```diff
@@ -261,3 +261,26 @@ func AssertNumberOfTaskruns(c *clients.Clients, namespace, numberOfTr, timeoutSe
 		assert.FailOnError(fmt.Errorf("Error: Expected %v number of taskruns but found %v number of taskruns", numberOfTr, len(trlist.Items)))
 	}
 }
+
+func AssertPipelinesExist(c *clients.Clients, status, namespace string) {
+	log.Printf("Verifying if pipelines %v in namespace %v", status, namespace)
+	pclient := c.Tekton.TektonV1beta1().Pipelines(namespace)
+	p, err := pclient.List(c.Ctx, metav1.ListOptions{})
+	if err != nil {
+		assert.FailOnError(fmt.Errorf("Error getting pipelines from namespace %v, Error: %v", namespace, err))
```

Comment:
The `else` statement is unnecessarily creates extra indentation. Doesn't `FailOnError` exit the function? If not, then return explicitly, if yes, no action is required, just omit `else` and type all commands inside the block directly.